packages/knex/src/index.ts

```diff
@@ -108,18 +108,23 @@ class DomainKnex {
       for (const key of domainSchema.keys()) {
         const column = decamelize(key);
         const value = domainSchema.values[key];
-        const type = value.type.constructor === Array ? value.type[0] : value.type;
+        const oneToMany = value.type.constructor === Array;
+        const type = oneToMany ? value.type[0] : value.type;
```

Comment:
Don't use `oneToMany` here, you are loosing readability of this code line, instead it this code line should be as it was before:
`const type = value.type.constructor === Array ? value.type[0] : value.type;`